code/game/objects/items/weapons/landmines.dm

```diff
@@ -77,9 +75,21 @@
 	else
 		..()
 
-/obj/item/landmine/attackby(var/obj/item/I, var/mob/user)
+/obj/item/landmine/attackby(obj/item/I, mob/user)
 	..()
-	if(I.force > 10 && deployed)
+	if(deployed && istype(I, /obj/item/wirecutters))
+		var/obj/item/wirecutters/W = I
+		user.visible_message(SPAN_WARNING("\The [user] starts defusing \the [src] with \the [W]." \
+							SPAN_NOTICE("You start defusing \the [src] with \the [W].")))
```

Comment:
Maybe add some flavor to tense moments like these? Could say something like, `[user] starts snipping some wires in \the [W] with \the [src]...` or something.

Reply:
donezo